html/pfappserver/root/static.alt/src/views/Nodes/_store/index.js

```diff
@@ -17,7 +18,8 @@ const state = {
   searchSortBy: 'mac',
   searchSortDesc: false,
   searchMaxPageNumber: 1,
-  searchPageSize: localStorage.getItem(STORAGE_SEARCH_LIMIT_KEY) || 10
+  searchPageSize: localStorage.getItem(STORAGE_SEARCH_LIMIT_KEY) || 10,
+  visibleColumns: localStorage.getItem(STORAGE_VISIBLE_COLUMNS_KEY) || false
```

Comment:
Since the localStorage will return a string, you'll need to wrap the value with `JSON.parse`.